recipes/qt/6.x.x/conanfile.py

```diff
@@ -0,0 +1,541 @@
+import os
+import shutil
+import itertools
+import glob
+
+import configparser
+from conans import ConanFile, tools, RunEnvironment, CMake
+from conans.errors import ConanInvalidConfiguration
+from conans.model import Generator
+
+
+class qt(Generator):
+    @property
+    def filename(self):
+        return "qt.conf"
+
+    @property
+    def content(self):
+        return "[Paths]\nPrefix = %s\n" % self.conanfile.deps_cpp_info["qt"].rootpath.replace("\\", "/")
+
+
+class QtConan(ConanFile):
+    _submodules = ["qtsvg", "qtdeclarative", "qttools", "qttranslations", "qtdoc",
+    "qtwayland","qtquickcontrols2", "qtquicktimeline", "qtquick3d", "qtshadertools", "qt5compat"]
+
+    generators = "pkg_config", "cmake_find_package", "cmake"
+    name = "qt"
+    description = "Qt is a cross-platform framework for graphical user interfaces."
+    topics = ("conan", "qt", "ui")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.qt.io"
+    license = "LGPL-3.0"
+    exports = ["patches/*.diff"]
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = {
+        "shared": [True, False],
+        "commercial": [True, False],
+
+        "opengl": ["no", "desktop", "dynamic"],
+        "with_vulkan": [True, False],
+        "openssl": [True, False],
```

Comment:
naming convention?

Reply:
we have the choice between [consistency with qt 5 recipe](https://github.com/conan-io/conan-center-index/blob/master/recipes/qt/5.x.x/conanfile.py#L46), and consistency with other options options. I have no personal preference. What do you prefer ?